### PR TITLE
Add reducers and actions

### PR DESCRIPTION
**Pull request by iamchristianani**
## Add Reducers and Actions for the Bookstore App

> I was able to complete the following:

- Installed all the required toolkits for Redux.
- Configure the App to make use of Redux.
- Created and Added the Books reducer.
- Created and Added the Categories reducer.
- Made use of the Ducks pattern.
- Fixed all Linter errors.

### CODE REVIEW

**Comment by DammyShittu** [comment]
Hi @iamchristianani,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### All the menu links in the Table of Contents should be present and working. The links from "Getting Started" to "Deployment" are not working. Kindly add them and update them with related information for a more professional documentation

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```
Comment (gemmen29) [inline src/redux/books/books.js]: - [ ] Kindly stick with the name convention of the [duck pattern](https://github.com/erikras/ducks-modular-redux) for naming actions like:
```javascript
<app name>/<reducer name>/<action type>
```
for example:
```javascript
const REMOVE_BOOK = 'bookstore/book/REMOVE_BOOK';
```

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const STATUS_CHECKED = 'STATUS_CHECKED';
```
Comment (gemmen29) [inline src/redux/categories/categories.js]: - [ ] Kindly stick with the name convention of the [duck pattern](https://github.com/erikras/ducks-modular-redux) for naming actions also here.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @iamchristianani, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @iamchristianani , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good PR and Description
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**